.github/workflows/test.yaml

```
@@ -40,8 +46,11 @@ jobs:
           fetch-depth: 0
       - name: dev env setup
         uses: ./.github/actions/dev-env-setup
-      - name: Run Nx Tests
-        run: yarn nx run-many -t test --skipNxCache
+      - name: Run Nx Affected Tests with Remote Caching
+        id: nx-tests
+        run: |
+          yarn nx affected:build --base=origin/develop --target=test --parallel
```

```suggestion
          yarn nx affected --base=origin/develop --target=test --parallel
```

The `something:other` format is for `nx run project:target`. Nx's affected is it's own thing akin to run, so you can drop it here. Target is defined as a `--target` attribute like you already had.